src/ui/developerMenu/ViewDeveloperMenu.qml

```diff
@@ -28,268 +28,245 @@ Item {
         id: flickableContent
 
         anchors.top: menu.bottom
-        anchors.topMargin: VPNTheme.theme.windowMargin
         anchors.left: parent.left
         anchors.right: parent.right
         height: parent.height - menu.height
 
-        VPNCheckBoxRow {
-            id: developerUnlock
+        ColumnLayout {
+            id: layout
 
-            width: parent.width - VPNTheme.theme.windowMargin
+            anchors.right: parent.right
+            anchors.left: parent.left
 
-            labelText:  VPNl18n.SettingsDevShowOptionTitle
-            subLabelText: VPNl18n.SettingsDevShowOptionSubtitle
+            spacing: 0
 
-            isChecked: VPNSettings.developerUnlock
-            onClicked: VPNSettings.developerUnlock = !VPNSettings.developerUnlock
-        }
+            onImplicitHeightChanged: {
```

Comment:
Nit: let's pad this number or add something like a `VPNVerticalSpacer{}` at the end of this `layout` so that there is a bit more of a margin between the `Qt version:` info text and the app window bottom. As is, I think this text will get covered up on iOS by the iPhone Home bar which is helpful for QA.